pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -338,12 +338,6 @@ func GetContainerLogs(client DockerInterface, pod *v1.Pod, containerID kubeconta
 }
 
 var (
-	// ErrNoContainersInPod is returned when there are no containers for a given pod
-	ErrNoContainersInPod = errors.New("NoContainersInPod")
-
-	// ErrNoPodInfraContainerInPod is returned when there is no pod infra container for a given pod
-	ErrNoPodInfraContainerInPod = errors.New("NoPodInfraContainerInPod")
```

Comment:
@yujuhong ok to remove these error reasons, or were you planning on using them?
Seemed usage of `ErrNoContainersInPod` stopped in bb417e82d7e40bb94d1928a01e06b8d7b1e6fa00 -- `pkg/kubelet/dockertools/manager.go` and `ErrNoPodInfraContainerInPod` doesn't look like it was ever in use as far as I can tell (note I haven't searched back before the origin of `pkg/kubelet/dockertools/manager.go`).